operators/Makefile

```diff
@@ -170,7 +177,10 @@ generate-all-in-one:
 
 # Deploy an all in one operator against the current k8s cluster
 deploy-all-in-one: GO_TAGS ?= release
-deploy-all-in-one: docker-build docker-push
+deploy-all-in-one:
+ifeq ($(SKIP_DOCKER_COMMAND), false)
+	$(MAKE) docker-build docker-push
+endif
```

Comment:
Why do we update `deploy-all-in-one`? AFAIU it's not used by `e2e`.

Reply:
Fixed